src/DropdownStateMixin.js

```diff
@@ -56,7 +57,7 @@ const DropdownStateMixin = {
   },
 
   bindRootCloseHandlers() {
-    let doc = React.findDOMNode(this).ownerDocument;
+    let doc = domUtils.ownerDocument(React.findDOMNode(this));
```

Comment:
You are repeating `domUtils.ownerDocument(React.findDOMNode(this))` a lot. Have you considered moving `React.findDOMNode(this)` inside the `domUtils.ownerDocument` function? So, you'd only have to call `domUtils.ownerDocument(this)`.